### ISSUE TITLE
Integrate backoff retry with Vert.x Circuit Breaker

### ISSUE THREAD

**Issue by sczyh30** [sczyh30/vertx-kue#3]
This is a good chance to introduce how and where to use **Vert.x Circuit Breaker**.

**Comment by pmlopes** [comment]
That could be interesting but if i'm not mistaken circuit breaker should be the main focus of other blueprint... i think if you're complete with the other one you could improve here

**Comment by sczyh30** [comment]
Yeah, Vert.x Circuit Breaker is the main focus of microservice blueprint.

At present I've implemented a basic retry backoff mechanism (with two strategies) and i think i could improve this with circuit breaker soon :-)
